cover-checker-console/src/main/java/com/naver/nid/cover/util/ParameterParser.java

```diff
@@ -48,17 +50,30 @@ public Parameter getParam(String... commandArgs) {
 			return null;
 		}
 
+		final List<String> coveragePaths = Arrays.asList(cmd.getOptionValues("c"));
+		final String[] coverageTypesArray = cmd.getOptionValues(COVERAGE_TYPE_OPTION);
+		List<String> coverageTypes = Arrays.asList(
+				coverageTypesArray == null
+						? new String[]{ "jacoco" }
+						: coverageTypesArray);
+
+		if ((coverageTypes.size() == 1) && coveragePaths.size() > 1) {
+			// Broadcast coverage type to all coverage paths
+			final String coverageType = coverageTypes.get(0);
+			coverageTypes = coveragePaths.stream().map(path -> coverageType).collect(Collectors.toList());
+		}
+
 		Parameter param = Parameter.builder()
```

Comment:
nit: if you wanted to be defensive you could wrap `coverageTypes` and `coveragePaths` in an modifiable list: `coverageTypes = Collections.unmodifiableList(coverageTypes);`

Reply:
This is done elsewhere in the code, I think it matches stylistically.